unity/Assets/Scripts/ARFlowDeviceSample.cs

```diff
@@ -25,16 +26,16 @@ public class ARFlowDeviceSample : MonoBehaviour
     private Vector2Int _sampleSize;
     private bool _enabled = false;
 
-    public TMP_InputField inputField;
+    public TMP_InputField ipField;
+    public TMP_InputField portField;
 
     // Start is called before the first frame update
     void Start()
     {
-        // const string serverURL = "http://192.168.1.100:8500";
-        // const string serverURL = "http://169.254.189.74:8500";
-        // const string serverURL = "http://192.168.1.139:8500";
-        const string serverURL = "http://192.168.1.219:8500";
-        inputField.text = serverURL;
+        const string initServerURL = "192.168.1.219";
```

Comment:
Will these default values be displayed on the screen? If so, we should use text placeholders like `server address` and `server port`

Reply:
Oh yeah, I'll add that

Reply:
Oh do you want to use the placeholders instead of default values? Or should I display the placeholders when the default values are used?

Reply:
Display placeholders when default values are used